README.md

```diff
@@ -116,29 +116,32 @@ Please note that there is no formal support for code from a nightly release, but
 
 This section gives a highlight of noteworthy changes.
 
-Refer to the [CHANGELOG](https://github.com/contentauth/c2pa-rs/blob/main/CHANGELOG.md) for detailed changes derived from git commit history.
+Refer to the [CHANGELOG](https://github.com/contentauth/c2pa-rs/blob/main/CHANGELOG.md) for detailed changes derived from Git commit history.
```

Comment:
Common misconception: "Git" should be capitalized. Please refer to git in lowercase as is preferred usage for this tool.

Reply:
No, Git is a proper noun, and is thus capitalized.  However `git` the command is not. 

Supporting references:
- https://git-scm.com/ This is the website for the project and is thus definitive. (n.b. the home page has "Git" only as the first word in a sentence and so is not a helpful guide, but elsewhere on the site it's consistently capitalized).
- https://english.stackexchange.com/questions/611711/tech-related-should-i-capitalize-the-word-git-in-this-context-or-not
- - https://github.com/swcarpentry/git-novice/issues/505
- https://en.wikipedia.org/wiki/Git

Reply:
This sounds correct to me